src/endpoints/groups.js

```diff
@@ -43,7 +43,7 @@ const deleteGroupLogo = async (req, res) => {
   authz.assertAuthorized(req.user, authz.actions.Write, req.context.group);
 
   const method = "DELETE";
-  const url = await req.context.group.source.urlFor("group-logo.png", method);
+  const url = await req.context.group.source.urlFor(req.context.group.source.logoFile, method);
```

Comment:
> Part of that weirdness comes from the `x.y.z.foo(x.y.z.bar)`

We could use new methods `urlForLogo()`/`urlForOverview()` to avoid this, but that probably goes deeper into the "no man's land".

> using `urlFor()` + a well-known string

I'd be more ok with the "well-known string" if it was used locally in just one file. My main motivation is to avoid duplicate hard-coding of this value across different files. What if we make it a `const` exported from this file?

> we could just as easily be calling this object in S3 `group-logo` with no extension

Is this common practice? I've not had to deal with supporting various image formats in storage via API before, but it kinda makes sense. Should this also be applied to `group-overview`, in case we decide to allow other text formats later on?

Reply:
Ok, in that case I plan to drop 352603ae53c37e7f93c2c4c022999addfe9f35f5 and continue with `group-logo.png` since that's used elsewhere in the code already. To support more image types in the future, it'd be good to store in S3 without the extension, but that requires more careful thought for proper transition. Splitting off as a separate issue: #594